dotnet-authserver/src/TeacherIdentity.AuthServer/Services/DqtApi/DqtApiClient.cs

```diff
@@ -56,4 +56,12 @@ public async Task PostTeacherNameChange(TeacherNameChangeRequest request, Cancel
         var response = await _client.PostAsync("/v3/teachers/name-changes", content, cancellationToken);
         response.EnsureSuccessStatusCode();
     }
+
+    public async Task PostTeacherDateOfBirthChange(TeacherDateOfBirthChangeRequest request,
```

Comment:
Is this method used anywhere?

Reply:
Yep used in Confirm.cshtml.cs above

Reply:
🙈 I read this as part of `Confirm.cshtml.cs`